lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -311,16 +321,18 @@ export default function BasicLTILaunchApp() {
 
   const content = (
     <div
-      className={classnames('BasicLTILaunchApp__content', {
-        'is-hidden': !contentReady || errorState,
+      className={classnames('flex flex-col h-full', {
+        invisible: !showContent,
+        visible: showContent,
```

Comment:
Stylistic alternative:

```suggestion
        showContent ? 'visible' : 'invisible'
```